### DISCUSSION TITLE
Honeypot JS 404

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#462] [_Discarded pitches]
### Current State
The Honeypot JS is running into an 404 Error.

GET https://testinstanz-fal.onofficeweb.com/wp-content/plugins/onoffice-for-wp-websites/onoffice-honeypot.js?ver=3588465d51e1e55d8a2235f9c115a182 net::ERR_ABORTED 404

GET https://testinstanz-mr.onofficeweb.com/wp-content/plugins/onoffice-for-wp-websites/onoffice-honeypot.js?ver=e8b1d4ecd58e03438cde6c9c1b0bbe0d net::ERR_ABORTED 404

### Desired State
The JS has to be accessible for the honeypot to work.

![404error](https://user-images.githubusercontent.com/88546396/224660805-2c75602b-2c85-4168-b98a-8f69dd17e65f.PNG)
![refused](https://user-images.githubusercontent.com/88546396/224660800-15d0f086-7000-45cc-9b50-04ab6e42ccd1.PNG)

**Comment by fredericalpers** [comment]
@dai-eastgate please investigate and give us an time estimate of how long you'll need to fix it.

**Reply by dai-eastgate** [reply]
Yes, I will check it now

**Reply by dai-eastgate** [reply]
@fredericalpers I need "half a day" to fix and "one day" to test it.
Because this "onoffice-honeypot.js" file will be loaded in "all forms" except "the application search" form.

**Reply by fredericalpers** [reply]
Please proceed. :)

**Comment by dai-eastgate** [comment]
@fredericalpers I fixed this bug, please check this video and give me your feedback
https://files.fm/u/6w8hwnjg3

**Reply by fredericalpers** [reply]
Thank you! I will consult with our devs on how we will proceed from here.

**Comment by dai-eastgate** [comment]
Merged in https://github.com/onOffice-Web-Org/oo-wp-plugin/pull/466
